src/styles/blocks/watch.scss

```
@@ -0,0 +1,55 @@
+.watch {
+  position: relative;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 500px;
+  width: 500px;
+  border-radius: 50%;
+  background-color: red;
+
+  &__point {
+    width: 20px;
+    height: 20px;
+    background-color: white;
+    border-radius: 50%;
+  }
+
+  &__seconds {
+    position: absolute;
+    top: 15px;
+    height: 230px;
+    width: 10px;
+    background-color: white;
+    border-radius: 20%;
+    animation-name: watch;
+    animation-duration: 60s;
+    animation-iteration-count: infinite;
+    animation-timing-function: linear;
```

```suggestion
    animation-name: watch;
    animation-duration: 60s;
    animation-iteration-count: infinite;
    animation-timing-function: linear;
```
You could use shorthand for this